app/script/view_model/bindings/CommonBindings.js

```diff
@@ -130,64 +130,69 @@ ko.bindingHandlers.resize = {
   init(element, valueAccessor, allBindings, data, context) {
     const params = ko.unwrap(valueAccessor()) || {};
 
-    const callback = params.callback;
-    const syncElement = document.querySelector(params.syncElement);
```

Comment:
We can safely remove those as this binding is now only used for a simple textarea